src/sentry/conf/server.py

```diff
@@ -728,7 +731,7 @@ def create_partitioned_queues(name):
         "important_django_request": {
             "()": "sentry.logging.handlers.MessageContainsFilter",
             "contains": ["CSRF"],
-        },
+        }
```

Comment:
What happened to our dangling commas? Does `black` remove them now?

Reply:
Not sure, maybe my local version of black is out of date or something.